lib/manager/git-submodules/extract.ts

```diff
@@ -106,7 +106,8 @@ export default async function extractPackageFile(
         try {
           const [currentValue] = (await git.subModule(['status', path]))
             .trim()
-            .split(/[+\s]/);
+            .replace(/^[-+]/, '')
```

Comment:
Is this fixing any open issue or just something you discovered?

Reply:
There isn't an open issue for this, but it does fix a bug